src/components/MenuAction.tsx

```diff
@@ -80,10 +77,12 @@ export const MenuAction = ({ items, changeMenu, menuIdx }: MenuActionProps) => {
     setId(anchorEl ? 'simple-popover' : undefined);
   }, [anchorEl]);
 
+  const item = actions.find(({ type }) => type === action);
+
   return (
     <>
       <ButtonBase className={`${classes.item} current`} onClick={handleClick}>
-        {items[menuIdx]?.title || '-'}
+        {item!.title || '-'}
```

Comment:
Why the exclamation mark? Why not question mark. Is `item = undefined` a case that should be allowed